src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+
+### name
+
+The human-readable label for the app shortcut when
+displayed to the user.
+
+### short_name (optional)
+
+The human-readable label used where space is limited. It is recommended
+that you provide it, even though it's
+optional.
+
+### description (optional)
+
+The human-readable purpose for the app shortcut.
+It is not used at the time of writing.
+
+### url
+
+The URL opened when a user activates the app
+shortcut. This URL must exist within the scope of the web app manifest. If it is
+a relative URL, the base URL will be the URL of the web app manifest.
+
+### icons (optional)
+
+An array of image resource objects. Each object must
+include the `src` and a `sizes` property. The `type` of image is optional.
+
+SVG files are not supported at the time of writing, use PNG instead.
+
+If you want pixel-perfect icons, provide them in
+increments of 48dp. Otherwise, it is recommended that you use a 192x192 pixels
+icon.
+
+## Test your app shortcuts
+
+To verify your app shortcuts are setup correctly, use the **Manifest** pane in the
+**Application** panel of DevTools.
+
+<figure class="w-figure">
+  <img src="./app-shortcuts-devtools.png" alt="Screenshot of app shortcuts in DevTools">
+  <figcaption class="w-figcaption">App shortcuts shown in DevTools</figcaption>
+</figure>
+
+This pane provides a human-readable version of many of your manifest's
+properties, including app shortcuts. It makes it easy to verify that all of the
+shortcut icons, if provided, are loading properly.
+
+{% Aside %}
+DevTools support for app shortcuts is available in Chrome 84 and Edge 84 and later.
+{% endAside %}
+
+App shortcuts may not be available right away to all users because PWA updates
+are capped to once a day.  You can find more details in this [article].
+
+## Best practises
+
+### Order app shortcuts by priority
+
+You are encouraged to order app shortcuts by priority, with the most critical
+app shortcuts appearing first in the `shortcuts` array as the limit on the
+number of app shortcuts displayed varies depending on the platform. Chrome and
+Edge on Windows for instance limit the number of app shortcuts to 10 while
+Chrome for Android only takes the first 4 app shortcuts into account.
+
+### Use distinct app shortcut names
+
+You should not rely on icons to differentiate app shortcuts as they may not
+always be visible. For instance, macOS doesn't support icons in the dock
+shortcuts menu. Use distinct names for each app shortcut.
+
+### Measure app shortcuts usage
+
+You should annotate app shortcuts `url` entries like you would do with
+`start_url` for analytics purposes (.e.g. `url:
+"/my-shortcut?utm_source=homescreen"`).
+
+## Browser support
+
+App shortcuts are available in Chrome 84 on Android. On Windows, you can
+experiment with them on both Chrome 84 and Edge 84 by enabling the flag
+`about://flags/#enable-desktop-pwas-app-icon-shortcuts-menu`. More desktop
+platform support will follow.
```

Comment:
I would much prefer this to be the first section of the guide. I know some of our contributors feel that it messes up the flow of the content, but we know from the MDN DNA results that cross-browser compatibility is web developers' number 1 frustration so I view it as a significant show of respect to them to put this information upfront. In other words, for some (most?) developers, when they realize that it's Chromium-only, they will stop reading. Therefore putting it upfront respects the reader's time.

Reply:
It is already at the very top of the article on purpose.
![image](https://user-images.githubusercontent.com/634478/82090332-a9e91480-96f5-11ea-84ac-1b50b5fcebed.png)

Reply:
Fair enough, thanks.